section2/reflection.md

```diff
@@ -1,15 +1,36 @@
+
+---
 
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
+- At first I found it frustrating, but it forced me to look up things I was uncertain about, and to use them in different ways, which helps me learn them better. It made me think outside of the copy/paste box, which forced me to type more and therefore learn more.
+
+---
 
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
+- A conditional Statement means the second part of the code is dependent on the first part. IF the first condition is met, then the code executes a statement. However if that condition is not met, then either nothing happens, or something different is executed.
+- An example of a real life conditional for me is if I don't take my vitamins, I will be really tired. If I do take my vitamins, I will not be near as tired.
+- An example in a web application is any ad you click on and try to learn more, they will only give you the information if you give your email. If you don't give your email, you don't get any info.
+
+---
 
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+// One way to do it is to use logical operators. With the logical operator && (and), both parts of the condition need to be true to have the condition go into affect. With the logical operator || (or), as long as one part of the condition is true, the condition will go into affect.
+
+---
 
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+//The logical operators will be helpful for this. The && (and) will make sure two conditions are met, while the || (or) will make sure one of the two conditions are met.
+
+---
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+7. What questions do you still have about `if` statements and/or functions?
+//I had trouble with else if statements. On some, the code would not read them as one command, and I'm not sure why. Does it have to do with the setup of my original if statement? Or bracket placement? Also, outside of true and false, I do not understand how the logical operators work.cd 
```

Comment:
If you can reference a specific line/lines of code and tell me what's going on with them (how do you know it's not readying them as one command?) - I'll be happy to look into it!